Dark.json

```diff
@@ -22341,6 +22341,7 @@
         "power": "2",
         "ability": "1",
         "deploy": "1",
+        "forfeit": "½",
```

Comment:
Silly Greedo. Breaking everything as usual.

Similar to the situation with destiny values... do we need to introduce a `forfeitValue` property?

Reply:
I kinda think, maybe?  That would be consistent.  Are there other characters with fractional forfeit values?  

If it's just Greedo, maybe setting it to be 0.5 would be better just for simplicity's sake, until we have a second?

Reply:
I think I could go either way with this one.

Reply:
Only Greedo has fractional forfeit.  I am not sure if this is relevant but Septoid Droid has asterisk forfeit and Brainiac has X forfeit.

Reply:
So if we went the "forfeitValue" property route, Greedo would have 0.5, Braniac wouldn't have anything, and septoid droid would have a 7, I think.

Reply:
Much like the destiny situation, I think it's good and important to have a field that displays the printed value. So, I'm leaning toward introducing a `forfeitValue` property. Again, since `forfeit` is a string, it's a good way to leverage that.

Reply:
Reverted to `0.5`.